Index.html

```diff
@@ -0,0 +1,112 @@
+        </nav>
+    </header>
+    <section class="sectionContainer">
+
+        <h1>Elio Cortés</h1>
+        <div>
+            <h2>Software Developer</h2>
+        </div>
+
+        <div>
+            <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="vector">
+            <img src="./images/Vector4.png" alt="vector">
+        </div>
+        <article class="experience">
+            <section class="box">
+                <h2>Language</h2>
+                <ul>
+                    <li>
+                        <p>Javascript</p>
+                    </li>
+                    <li>
+                        <p>Ruby</p>
+                    </li>
+                    <li>
+                        <p>Html</p>
+                    </li>
+                    <li>
+                        <p>Css</p>
+                    </li>
+                </ul>
+            </section>
+            <section class="box">
+                <h2>Frameworks</h2>
+                <ul>
+                    <li>
+                        <p>Bootstrap</p>
+                    </li>
+                    <li>
+                        <p>Ruby on Rails</p>
+                    </li>
+                    <li>
+                        <p>RSPec</p>
+                    </li>
+                    <li>
+                        <p>CapyBara</p>
+                    </li>
+                    <li>
+                        <p>Selenium</p>
+                    </li>
+
+                </ul>
+            </section>
+            <section class="box">
+                <h2>Skills</h2>
+                <ul>
+                    <li>
+                        <p>Codekit</p>
+                    </li>
+                    <li>
+                        <p>Github</p>
+                    </li>
+                    <li>
+                        <p>Codepen</p>
+                    </li>
+                    <li>
+                        <p>Gitlab</p>
+                    </li>
+                    <li>
+                        <p>Terminal</p>
+                    </li>
+                </ul>
+            </section>
+        </article>
+        <div class="vector">
+            <img src="./images/Vector4.png" alt="vector">
+        </div>
+        <div class="socialContent">
+            <ul>
+                <li><a href="#"><img src="./icons/github.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./icons/linkedIn.png" alt="linkedIn icon"></a></li>
+                <li><a href="#"><img src="./icons/angelist.png" alt="angelist icon"></a></li>
+                <li><a href="#"><img src="./icons/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./icons/medium.png" alt="medium icon"></a></li>
+            </ul>
+        </div>
```

Comment:
[OPTIONAL] It would be better to align the social media icons to the center of the page to improve your UI design.
<img width="210" alt="image" src="https://user-images.githubusercontent.com/77190203/154529941-a6aaaa98-3f0c-475c-90d1-c3ad24c6e9c7.png">

Reply:
thanks @medaminedev66 I’ll keep that in mind for my next delivery